migration/sql-files/080-remove-unknown-link-types.sql

```diff
@@ -0,0 +1,95 @@
+SET LOCAL linktypes.bug_blocker = '{{index . 0}}';
+SET LOCAL linktypes.related = '{{index . 1}}';
+SET LOCAL linktypes.parenting = '{{index . 2}}';
+
+SET LOCAL linkcats.systemcat = '{{index . 3}}';
+SET LOCAL linkcats.usercat = '{{index . 4}}';
+
+-- The following link types exist in the current production database but they
+-- are not known to the code and therefore we re-assign all links associated to
+-- those link types with their appropriate link type and later remove these
+-- unknown link types.
+SET LOCAL linktypes.unknown_bug_blocker = 'aad2a4ad-d601-4104-9804-2c977ca2e0c1';
```

Comment:
Didn't we find that there were other UUID's for the same categories in prod? 

Should we reverse this and update based on LinkType name instad, that should exclude other unknown.

Reply:
No, we had revisions pointing to non existing types wasn't it.. which is fixed by the update rev.type = link.type..

?